sxw_soilwat.c

```diff
@@ -143,11 +137,11 @@ static void _update_transp_coeff(void) {
         ForEachGroup(g)
         if (RGroup[g]->veg_prod_type == SW_SHRUB) {
             if (getNTranspLayers(SW_SHRUB))
-                y->transp_coeff[SW_SHRUB] += (RealF) _roots_max[Ilg(l, g)] * RGroup[g]->rgroupFractionOfVegTypeBiomass;
+                y->transp_coeff[SW_SHRUB] += (RealF) SXWResources->_roots_max[Ilg(l, g)] * RGroup[g]->rgroupFractionOfVegTypeBiomass;
 
             /*printf("* lyr=%d, group=%s(%d), type=%d, tl=%d, rootmax=%f, relsize2=%f, trco=%f\n",
               l, RGroup[g]->name, g, RGroup[g]->veg_prod_type, getNTranspLayers(RGroup[g]->veg_prod_type),
-              _roots_max[Ilg(l, g)], RGroup[g]->relsize, y->transp_coeff[SW_SHRUB]);
+              SXWResources->_roots_max[Ilg(l, g)], RGroup[g]->relsize, y->transp_coeff[SW_SHRUB]);
              */
```

Comment:
Can we get rid of this commented-out code?

Reply:
I would like to keep this (after we updated the references to relsize)